concept/type/Type.java

```diff
@@ -34,6 +34,8 @@
     @CheckReturnValue
     boolean isRoot();
 
+    boolean isRelationType();
```

Comment:
Add the others too

Reply:
Fixed in https://github.com/graknlabs/client-java/pull/211/commits/36f8dd8033f9cc19da804e18f78a5d347f40aa76